### ISSUE TITLE
compile error with gcc 10

### ISSUE THREAD

**Issue by cipig** [DigiByte-Core/digibyte#88]
compilation on debian 11 with gcc 10 fails
```
./autogen.sh
./configure --with-incompatible-bdb --with-gui=no --disable-tests --disable-bench --without-miniupnpc --disable-zmq --disable-wallet
make -j8
```
error
```
httpserver.cpp:75:10: error: ‘deque’ in namespace ‘std’ does not name a template type
   75 |     std::deque<std::unique_ptr<WorkItem>> queue;
      |          ^~~~~
httpserver.cpp:34:1: note: ‘std::deque’ is defined in header ‘<deque>’; did you forget to ‘#include <deque>’?
   33 | #include <support/events.h>
  +++ |+#include <deque>
   34 | 
httpserver.cpp: In member function ‘bool WorkQueue<WorkItem>::Enqueue(WorkItem*)’:
httpserver.cpp:93:13: error: ‘queue’ was not declared in this scope; did you mean ‘Enqueue’?
   93 |         if (queue.size() >= maxDepth) {
      |             ^~~~~
      |             Enqueue
httpserver.cpp:96:9: error: ‘queue’ was not declared in this scope; did you mean ‘Enqueue’?
   96 |         queue.emplace_back(std::unique_ptr<WorkItem>(item));
      |         ^~~~~
      |         Enqueue
httpserver.cpp: In member function ‘void WorkQueue<WorkItem>::Run()’:
httpserver.cpp:107:35: error: ‘queue’ was not declared in this scope; did you mean ‘Enqueue’?
  107 |                 while (running && queue.empty())
      |                                   ^~~~~
      |                                   Enqueue
httpserver.cpp:111:31: error: ‘queue’ was not declared in this scope; did you mean ‘Enqueue’?
  111 |                 i = std::move(queue.front());
      |                               ^~~~~
      |                               Enqueue
```

**Comment by cipig** [comment]
https://github.com/litecoin-project/litecoin/commit/ba8a8c46b3e929d118b57e30fe38cc48bd309557 fixes this

**Comment by JaredTate** [comment]
Thanks for brining this issue up, this issue is most likely fixed with v8.22 that is under active development. Cheers.
